logstash-plugins/logstash-filter-useragent
# Upgrade `snakeyml` dependency to 1.33

## What does this PR change?
Upgrades `snakeyml` to 1.33 to mitigate the security vulnerabilities.

### How to test locally?
- build jar with `./gradlew clean vendor`
    - creates logstash-filter-useragent-3.3.4.jar
- define a config to use the plugin
```
input {
  stdin {}
}
filter {
  useragent {
    source => 'message'
  }
}
output {
 stdout {}
}
```

- run Logstash and check if plugin is parsing the user agent info
```
Mozilla/5.0 (iPad; U; CPU OS 3_2_1 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Mobile/7B405
{
    "@timestamp" => 2023-03-11T04:46:48.174056Z,
          "host" => {
        "hostname" => "Mashhurs-MacBook-Pro.local"
    },
       "message" => "Mozilla/5.0 (iPad; U; CPU OS 3_2_1 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Mobile/7B405",
      "@version" => "1",
         "event" => {
        "original" => "Mozilla/5.0 (iPad; U; CPU OS 3_2_1 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Mobile/7B405"
    },
    "user_agent" => {
        "device" => {
            "name" => "iPad"
        },
          "name" => "Mobile Safari UI/WKWebView",
            "os" => {
               "name" => "iOS",
            "version" => "3.2.1",
               "full" => "iOS 3.2.1"
        }
    }
}
```
